cdc/owner.go

```diff
@@ -438,6 +440,16 @@ func (o *ownerImpl) handleMarkdownProcessor(ctx context.Context) {
 			remainProcs = append(remainProcs, snap)
 			continue
 		}
+		err = kv.DeleteTaskPosition(ctx, o.etcdClient, snap.CfID, snap.CaptureID)
+		if err != nil {
+			log.Warn("failed to delete processor info",
```

Comment:
```suggestion
			log.Warn("failed to delete task position",
```
similar suggestion in L435, s/processor info/task status